### PR TITLE
feat: create Routes, add Home and Form pages and add TailwindCSS

### PR DESCRIPTION
**Pull request by pilyct**

### CODE REVIEW

File: client-ts/src/vite-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (dinarabs) [inline client-ts/src/vite-env.d.ts:1]: can this be removed?

File: client-ts/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  theme: {
+    extend: {
+      colors: {
+        // Add your custom colors here
+        'dark-blue': '#0C2D42',
+        'blue': '#178BC8',
+        'dark-grey':'#413D3D',
+        'grey': '#9E9C98',
+        'light-grey':'#EEEEEE',
+        'light-fucsia':'#AD62D1',
```
Comment (dinarabs) [inline client-ts/tailwind.config.js:15]: pls add // TODO comment

File: client-ts/src/Routes/Routes.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import Home from '../pages/Home'
+import EntryForm from '../pages/EntryForm'
+import NotFound from '../pages/NotFound'
+
+const AppRoutes = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+          <Route path='/home' element={<Home/>}/>
+          <Route path='/form' element={<EntryForm/>}/>
```
Comment (dinarabs) [inline client-ts/src/Routes/Routes.tsx:11]: Maybe better name this `CreateEntryForm` or `CreateForm`

File: client-ts/src/assets/react.svg
Comment (dinarabs) [inline client-ts/src/assets/react.svg:1]: this can be deleted

File: client-ts/src/components/Controls.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { Link } from 'react-router-dom';
+
+const Controls: FC = () => {
+  return (
+    <div className='flex items-center justify-between shadow-md w-full my-8 h-24 p-4 border-2 border-light-grey rounded-lg'>
+      <Link className='w-3/12 bg-blue rounded h-12 text-center text-white' to='/form'>
+        <button className='h-12'>+Add</button>
+      </Link>
+      <input 
+        // value={cityName}
```
Comment (dinarabs) [inline client-ts/src/components/Controls.tsx:11]: pls add // TODO comment

File: client-ts/src/components/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+
+const Navbar: FC = () => {
+  return (
+    <nav className='flex items-center justify-center bg-dark-blue h-20 shadow-xl'>
+      <div className='w-6/12 flex items-center justify-between'>
+        <div className='flex items-center justify-center'>
+          <img src='../logo-sm.svg' className='inline w-16 h-16'/>
```
Comment (dinarabs) [inline client-ts/src/components/Navbar.tsx:8]: maybe in the future we can add navigation to home

File: client-ts/src/pages/EntryForm.tsx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+      <h2 className='text-bold text-4xl'>Create new Diaree entry</h2>
+      <form>
+        <div className='space-y-12'>
+          <div className='border-b border-gray-900/10 pb-12'>
+            <p className='mt-1 text-sm leading-6 text-gray-600'>Whether it's a cherished memory or a fleeting emotion, let the words flow freely and embrace the joy of reflection.</p>
+        
+            <div className='mt-4 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6'>
+
+              <div className='col-span-full'>
+                <label htmlFor='cover-photo' className='block text-sm font-medium leading-6 text-gray-900'>Cover photo</label>
```
Comment (dinarabs) [inline client-ts/src/pages/EntryForm.tsx:31]: this can be separate UploadImage component

**Review by dinarabs — APPROVED** [review]
LGTM 🚀 let's gooooooooo!

File: client-ts/src/vite-env.d.ts (hunk 1 above)
Comment (pilyct) [reply client-ts/src/vite-env.d.ts:1]: Not sure if I should remove it. I think we may discover it once we start implementing the functionality in the components. These what I found about it: 

**Vite Docs && Articles:**
- https://v2.vitejs.dev/guide/env-and-mode.html#intellisense-for-typescript
- https://dev.to/rxliuli/vite-automatically-generates-type-definitions-for-environment-variables-44gh
- https://dev.to/yanagisawahidetoshi/mastering-environment-variables-in-vitetypescriptreact-1odi


**what ChatGPT told me:**
The `vite-env.d.ts` file is a TypeScript declaration file that is typically used to define types and interfaces for the environment variables used in a Vite project. In many TypeScript projects, including those built with Vite, it's common to have declaration files (files with a .d.ts extension) to provide type information for external modules, libraries, or in this case, environment variables.

The `vite-env.d.ts` file is not a standard or mandatory filename. Developers might create such a file to extend or augment the type information available for the environment variables used in their Vite project.

**Pull request merged**